openlibrary/templates/search/publishers.html

```diff
@@ -22,7 +22,7 @@ <h1>
                 <a href="$p.key">$p.name</a>
 
                 <span class="count">
-                    <b>$ungettext("1 book", "%(count)s books", p.count, count=commify(p.count))</b>
+                    <b>$ungettext("1 book", "%(count)d books", p.count, count=p.count)</b>
```

Comment:
@cdrini The use of `commify` is historical and I was under the impression from your comments early on that you wanted it left in. Why are these the only two instances to be removed?

Reply:
Ah, forgot to comment. It was causing an error, because some instances of "1 book" used `%s` in their plural, and others used `%d`. It seemed like more used `%d`, so I just chose to standardize around that.

Reply:
Are you sure there wasn't a mismatch between the code and the `.pot` or `.po` files? Those should be treated as two different msgids which are handled distinctly without problem.

Are like scenario is that the code was changed, the new msgids extracted to the `.pot`, an old translation was picked up and tagged with the `fuzzy` tag, then the `fuzzy` tag got removed without noticing that the %d had changed to %s.

Having said that, I'm happy to get rid of all the `commify()`s. I always thought they were superfluous.